### PR TITLE
Fwdport/7.4 beta master

### PR DESCRIPTION
**Pull request by alexanderchan-scality**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "engines": {
       "node": "6.9.5"
   },
-  "version": "7.0.0",
+  "version": "7.4.0-beta",
```
Comment (DavidPineauScality) [inline package.json]: beware of the version change & dependency change

File: package.json [hunk 2]
```diff
@@ -17,18 +17,18 @@
   },
   "homepage": "https://github.com/scality/utapi#readme",
   "dependencies": {
-    "arsenal": "scality/Arsenal",
+    "arsenal": "scality/Arsenal#rel/7.4-beta",
```
Comment (DavidPineauScality) [inline package.json]: Branch spec needs to be removed for master

File: package.json [hunk 3]
```diff
@@ -17,18 +17,18 @@
...
     "async": "^2.0.1",
     "ioredis": "^2.3.0",
     "node-schedule": "1.2.0",
-    "vaultclient": "scality/vaultclient",
-    "werelogs": "scality/werelogs"
+    "vaultclient": "scality/vaultclient#rel/7.4-beta",
```
Comment (DavidPineauScality) [inline package.json]: Branch spec needs to be removed for master

File: package.json [hunk 4]
```diff
@@ -17,18 +17,18 @@
...
+    "werelogs": "scality/werelogs#rel/7.4-beta"
```
Comment (DavidPineauScality) [inline package.json]: Branch spec needs to be removed for master

File: package.json [hunk 5]
```diff
@@ -17,18 +17,18 @@
...
   },
   "devDependencies": {
     "eslint": "^2.4.0",
     "eslint-plugin-react": "^4.3.0",
     "eslint-config-airbnb": "^6.0.0",
-    "eslint-config-scality": "scality/Guidelines",
+    "eslint-config-scality": "scality/Guidelines#rel/7.4-beta",
```
Comment (DavidPineauScality) [inline package.json]: Branch spec needs to be removed for master

**Comment by rahulreddy** [comment]
Integration passed here http://ci.ironmann.io/gh/scality/Integration/20791

**Pull request merged**